tools/dev/dev.go

```diff
@@ -0,0 +1,171 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+)
+
+func main() {
+	var opts struct {
+		Node                   int    `long:"node" default:"1"`
```

Comment:
Ideally we could do "start 4 nodes" and it then spins up those 4 nodes. The config is exactly the same (besides the ports)

Reply:
The idea was to keep the script for exactly one node so you run it from 4 terminal tabs for example. This way, you can easily kill a node, restart a node. Switch the version of a node, etc. This helps debugging cluster issues.

Reply:
oki, then ignore :)

Reply:
@etiennedi could you give some feedback?